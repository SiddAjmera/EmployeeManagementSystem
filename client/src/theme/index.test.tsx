import { create } from "react-test-renderer";
import ThemeConfig from "./";

describe("ThemeConfig", () => {
  test("renders the ThemeConfig", () => {
    const tree = create(
      <ThemeConfig>
        <h1>Some Content</h1>
      </ThemeConfig>
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
