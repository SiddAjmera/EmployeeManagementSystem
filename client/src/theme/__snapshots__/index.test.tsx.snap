// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeConfig renders the ThemeConfig 1`] = `
<h1>
  Some Content
</h1>
`;
